python/nx-cugraph/README.md

```diff
@@ -2,31 +2,78 @@
 
 ## Description
 [RAPIDS](https://rapids.ai) nx-cugraph is a [backend to NetworkX](https://networkx.org/documentation/stable/reference/classes/index.html#backends)
-with minimal dependencies (`networkx`, `cupy`, and `pylibcugraph`) to run graph algorithms on the GPU.
+to run supported algorithms with GPU acceleration.
 
-### Contribute
+## System Requirements
 
-Follow instructions for [contributing to cugraph](https://github.com/rapidsai/cugraph/blob/branch-23.12
-and [building from source](https://docs.rapids.ai/api/cugraph/stable/installation/source_build/), then build nx-cugraph in develop (i.e., editable) mode:
-```
-$ ./build.sh nx-cugraph --pydevelop
-```
+Using nx-cugraph with this notebook requires the following:
+
+ * NVIDIA GPU, Pascal architecture or later
+ * CUDA 11.2, 11.4, 11.5, 11.8, or 12.0
+ * Python versions 3.9, 3.10, or 3.11
+ * NetworkX >= version 3.2
+
+More details about system requirements can be found in the [RAPIDS System Requirements documentation](https://docs.rapids.ai/install#system-req)..
+
+## Installation
 
-### Run tests
+nx-cugraph can be installed using either conda or pip.
 
-Run nx-cugraph tests from `cugraph/python/nx-cugraph` directory:
+### conda
 ```
-$ pytest
+conda install -c rapidsai-nightly -c conda-forge -c nvidia nx-cugraph
 ```
-Run nx-cugraph benchmarks:
+### pip
 ```
-$ pytest --bench
+python -m pip install nx-cugraph-cu11 --extra-index-url https://pypi.nvidia.com
```

Comment:
Does this need a `--pre` or `>=23.10.0a0` specifier since the latest changes? @vyasr may know. Maybe based on the note below about these being non-nightlies it's not an issue?

Reply:
yes, I was relying on the note below that since (I don't believe) we have a way to provide nightly 23.12 wheels. Although if there is a way I'm not aware of, I'd love to be able to describe how users can get nightly 23.12 wheels.  @vyasr any thoughts?